Mathlib/CategoryTheory/Yoneda.lean

```diff
@@ -142,6 +142,11 @@ def punitIso : coyoneda.obj (Opposite.op PUnit) ≅ 𝟭 (Type v₁) :=
 def objOpOp (X : C) : coyoneda.obj (op (op X)) ≅ yoneda.obj X :=
   NatIso.ofComponents fun _ => (opEquiv _ _).toIso
 
+/-- Taking the `unop` of morphisms is a natural isomorphism. -/
+def opOpYoneda : yoneda ⋙ (whiskeringLeft _ _ _).obj (opOp C) ≅ coyoneda :=
```

Comment:
As this is in the `Coyoneda` namespace, the name `opIso` might be slightly berrer.